pkg/util/workerqueue/workerqueue.go

```diff
@@ -37,6 +37,29 @@ const (
 	workFx = time.Second
 )
 
+// DebugError is a marker type for errors that that should only be logged at a Debug level.
+// Useful if you want a Handler to be retried, but not logged at an Error level.
+type DebugError struct {
```

Comment:
Is this type intended to be used outside of this package? Given that the constructor below returns `error` it seems like this should be package scoped and callers should use `NewDebugError` to create them.